hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+This patch reorder typed dicts processing in `hypothesis.strategies._internal.core._from_type`
+to make typed-dict based classes available for registering default stategy via `register_type_strategy`.
+
+There is no change to the public API or behaviour.
```

Comment:
```suggestion
This patch enables :func:`~hypothesis.strategies.register_type_strategy` for subclasses of 
:class:`python:typing.TypedDict`.  Previously, :func:`~hypothesis.strategies.from_type`
would ignore the registered strategy (:issue:`2872`).

Thanks to Ilya Lebedev for identifying and fixing this bug!
```

As well as tidying up and fixing the release notes (there *is* a change to public behaviour!), we like to credit our contributors 😁   Please add yourself to the list in `AUTHORS.rst` too!